nemo/core/optim/lr_scheduler.py

```diff
@@ -640,6 +640,119 @@ def _get_lr(self, step):
         return new_lrs
 
 
```

Comment:
Name of such class is by conversion "Annealing' at the end, not Policy. Though it's fine since it's just convention